openverse_catalog/dags/common/loader/smithsonian_unit_codes.py

```diff
@@ -61,74 +28,34 @@ def get_new_and_outdated_unit_codes(unit_code_set, sub_prov_dict=SUB_PROVIDERS):
     new_unit_codes = unit_code_set - sub_provider_unit_code_set
     outdated_unit_codes = sub_provider_unit_code_set - unit_code_set
 
-    if bool(new_unit_codes):
-        logger.info(
-            f"The new unit codes {new_unit_codes} must be added to "
-            f"the SMITHSONIAN_SUB_PROVIDERS dictionary"
-        )
-
-    if bool(outdated_unit_codes):
-        logger.info(
-            f"The outdated unit codes {outdated_unit_codes} must be "
-            f"deleted from the SMITHSONIAN_SUB_PROVIDERS dictionary"
-        )
-
     return new_unit_codes, outdated_unit_codes
 
 
 def alert_unit_codes_from_api(
-    postgres_conn_id,
-    unit_code_table="smithsonian_new_unit_codes",
     units_endpoint=UNITS_ENDPOINT,
     query_params=PARAMS,
 ):
+    """
+    Alert in Slack if human intervention is needed to update the
+    SMITHSONIAN_SUB_PROVIDERS dictionary
+    """
     response = requests.get(units_endpoint, params=query_params)
     unit_code_set = set(response.json().get("response", {}).get("terms", []))
     new_unit_codes, outdated_unit_codes = get_new_and_outdated_unit_codes(unit_code_set)
 
-    initialise_unit_code_table(postgres_conn_id, unit_code_table)
-
-    postgres = PostgresHook(postgres_conn_id=postgres_conn_id)
+    if bool(new_unit_codes) or bool(outdated_unit_codes):
+        message = "\n*Updates needed to the SMITHSONIAN_SUB_PROVIDERS dictionary*:\n\n"
```

Comment:
Since we're using `bool(<value>)` a few times, it might be worth doing assignment expressions here?

```suggestion
    if (has_new_codes := bool(new_unit_codes)) or (has_outdated_codes := bool(outdated_unit_codes)):
        message = "\n*Updates needed to the SMITHSONIAN_SUB_PROVIDERS dictionary*:\n\n"
```

And then use `has_<new|outdated>_codes` below

Reply:
I generally agree, but in this case I think if `has_new_codes` is True, then the conditional will immediately be evaluated to True and `has_outdated_codes` won't get assigned 🤔 Since we're only evaluating `bool(<value>)` twice per value, I'm not sure it's worth working around that.

Reply:
WOAH, okay I definitely didn't consider that! Good to know for the future :memo: